apps/mobile/src/utils/notifications/checkForNewOffers.ts

```diff
@@ -71,6 +69,9 @@ async function displayNotification(t: TFunction): Promise<void> {
     },
     android: {
       channelId: await getDefaultChannel(),
+      pressAction: {
```

Comment:
This is actually pretty interesting, this makes the press action work on every android device...